### DISCUSSION TITLE
Importing pole figure data from a Jordan Valley XRD tool

### DISCUSSION THREAD

**Discussion by CreativeFolios** [mtex-toolbox/mtex#1912] [Ask Anything]
Dear All,

I am new to MTEX Matlab Toolbox and trying to import pole figure data from a Jordan Valley XRD tool (about 10 years old). The data was provided to me, and I have no access to the tool's software. The data contains 36 files with .X01 extension and 1 file with .OC3 extension. .X01 files provide the data for Chi angle variation from 0 to 70° each one covering 2° portion, and I believe .OC3 file is the one the tool software reads to plot pole figure. .X01 files contain measurement details and dat as 2 column as tilt angle and intensity. Is there a way to import this data format using MTEX import function?

Thank you very much for your help in advance.

Best regards,

**Comment by kilir** [comment]
Hi,
can you provide an example file?
Cheers,
Rüdiger
